src/main/java/com/wanderersoftherift/wotr/init/WotrAttributes.java

```diff
@@ -24,6 +24,8 @@ public class WotrAttributes {
 
     private static final List<DeferredHolder<Attribute, RangedAttribute>> PLAYER_ATTRIBUTES = new ArrayList<>();
```

Comment:
With the change in usage would suggest renaming to just ATTRIBUTES (along with its usages. (shift + F6 in IntelliJ)